### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following:

- Added the mobile menu for the mobile version
- Added main.js file to activate the hamburger button on click.

### CODE REVIEW

**Comment by OmarRamoun** [comment]
Hi, @PNdunguMaina and @elhamatokhi. Can you please, fix the linter errors so that I can proceed with the review

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟠 Project Invalid! ⚓

## Hi @PNdunguMaina 👋

Unfortunately 😕, the project is *invalid* (that means I can not review it because you missed some rules that I have listed for you below ⬇️).

While you made a **really great** effort 💪 in this project that I really appreciate 🙇, unfortunately, _I cannot proceed to review your code_.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as **invalid** in your Dashboard. Don't worry though, You still have **3** More attepmts. So please submit a new one once you are ready 🙏.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -15,14 +15,24 @@
         <div class="navbar">
           <a href="#" class="logo">Patrick Maina</a>
           <nav class="side-nav">
+            
             <ul class="nav-items">
```
Comment (thecodechaser) [inline index.html:19]: - [ ] After opening the mobile menu and closing it I am not able to open it again until I refresh the page. Please make sure to fix this issue and users should be able to open it every time without refreshing the page.

File: index.html [hunk 2]
```diff
@@ -15,14 +15,24 @@
...
               <li class="item-1"><a class="item-name-1" href="#">Portfolio</a></li>
               <li class="item-2"><a class="item-name-2" href="#">About</a></li>
               <li class="item-3"><a class="item-name-3" href="#">Contact</a></li>
-              <li class="mail-icon"><a href="#"><img class="mail-bar" src="Union.svg" alt="Mail bar"></a></li></a></li>
+              <li class="mail-icon"><a href="#"><img class="mail-bar" src="Images/Union.svg" alt="Mail bar"></a></li></a></li>
             </ul>
```
Comment (thecodechaser) [inline index.html:24]: - [ ] For this project you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design as much as possible. Your design is missing the right arrow icons in front of menu options and the background images. Please make sure to include them in your design and your design should match the Figma template.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @PNdunguMaina, and @elhamatokhi👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PNdunguMaina** [comment]
Hello, kindly advise on how to solve the issue with the hamburger menu so
that it can be clickable all the time without refreshing the page.

**Review by muneebulrehman — APPROVED** [review]
Hi @PNdunguMaina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**